doc/source/getting_started/docker.rst

```diff
@@ -0,0 +1,343 @@
+
+.. code:: bash
+
+   docker start mapdl
+
+Or just delete it and run it again using:
+
+.. code:: bash
+
+    docker rm -f mapdl
+
+    docker run -e ANSYSLMD_LICENSE_FILE=$ANSYSLMD_LICENSE_FILE --restart always --name mapdl -p $LOCAL_MAPDL_PORT:50052   $MAPDL_DOCKER_REGISTRY_URL -smp > log.txt
+
+
+You can append the Docker flag ``--rm`` to automatically clean up the container
+when it exits.
+
+The preceding commands create a log file (``log.txt``) in your current directory location, you can
+remove the following part ``> log.txt `` to avoid this. In that case the command
+output is redirected to the console which is kept blocked until the Docker image exits.
+You can detach the console from the Docker container output by appending ``-d`` to the
+`docker run <docker_run_>`_  command (always add this before the Docker image URL).
+If you don't want to block the console, the best approach is to pipe the output to a file
+as mentioned before so you can inspect the output of that file.
+
+Notice that the MAPDL Docker image gRPC port (``50052``) is being mapped to a
+different host port (``50053``) to avoid port conflicts with local
+MAPDL instances running on the host or other Docker images.
+You could additionally launch more Docker containers in different ports if
+you want to run multiple simulations at the same time.
+The module :ref:`ref_pymapdl_pool`` does not work when you are connecting
+to a remote MAPDL Docker image so it does not work when connected to Docker
+containers either. If you decide to launch multiple MAPDL instance, you must
+manage these instances by yourself.
+
+.. note:: Ensure that your port ``50053`` is open in your local firewall.
+
+You can provide additional MAPDL command line parameters to MAPDL by simply
+appending them to the end of the command.
+For example, you can increase the number of processors (up to the
+number available on the host machine) with the ``-np`` switch:
+
+.. code:: bash
+
+  docker run -e ANSYSLMD_LICENSE_FILE=$ANSYSLMD_LICENSE_FILE --restart always --name mapdl -p $LOCAL_MAPDL_PORT:50052 $MAPDL_DOCKER_REGISTRY_URL -smp -np 8 > log.txt
+
+
+For additional command line arguments visit the *Notes* section
+within :func:`launch_mapdl() <ansys.mapdl.core.launch_mapdl>`.
+
+You can use a script file (batch ``'.bat'`` or PowerShell ``'.ps'``)
+to run the preceding commands all at once.
+
+Once you have launched MAPDL you should see in your console (or output file):
+
+.. code::
+
+    Start GRPC Server
+
+    ##############################
+    ### START GRPC SERVER      ###
+    ##############################
+
+    Server Executable   : MapdlGrpc Server
+    Server listening on : 0.0.0.0:50052
+
+
+.. warning:: 
+  
+   You should notice that the port specified in the console is the internal Docker container port.
+   This port has been mapped to the value specified in :env:`LOCAL_MAPDL_PORT`.
+
+
+.. _run_an_mapdl_image_using_docker_compose:
+
+Using ``docker-compose`` to launch MAPDL
```

Comment:
@germa89 I only edited to this point and had to leave for an event. Let me know if you want me to continue with edits tomorrow.

Reply:
Hi @PipKat  please do :)